### PR TITLE
[RM-Ch02 ] Change VNF to Workloads

### PR DESCRIPTION
**Pull request by ASawwaf**
Change VNF to Workloads [VNFCs/Micro-service]
for digrams, I need your support @rabi-abdel @kedmison

### CODE REVIEW

**Comment by pgoyal01** [comment]
@ASawwaf:
1. For consistency, should "VNFs/CNFs (workloads)" be reworded as "workloads (VNFs/CNFs)" as in the first paragraph of the chapter:

The Cloud Infrastructure is the totality of all hardware and software components which build up the environment in which VNFs/CNFs (workloads) are deployed, managed and executed. It is, therefore, inevitable that different workloads would require different capabilities and have different expectations from it.

1. Change The Figure legend: "Figure 2-1: Infrastructure profiles proposed based on VNFs categorisation." to "Figure 2-1: Infrastructure profiles proposed based on workloads (VNFs/CNFs) categorisation."

**Comment by ASawwaf** [comment]
> @ASawwaf:
> 
> 1. For consistency, should "VNFs/CNFs (workloads)" be reworded as "workloads (VNFs/CNFs)" as in the first paragraph of the chapter:
> 
> The Cloud Infrastructure is the totality of all hardware and software components which build up the environment in which VNFs/CNFs (workloads) are deployed, managed and executed. It is, therefore, inevitable that different workloads would require different capabilities and have different expectations from it.
> 
> 1. Change The Figure legend: "Figure 2-1: Infrastructure profiles proposed based on VNFs categorisation." to "Figure 2-1: Infrastructure profiles proposed based on workloads (VNFs/CNFs) categorisation."


@pgoyal01 due to the fact the here we initially mention VNFC so I used microservices to corresponding to VNFC, but I ok to change all to "workloads (VNFs/CNFs)"

**Comment by CsatariGergely** [comment]
> 
> 
> > @ASawwaf:
> > 
> > 1. For consistency, should "VNFs/CNFs (workloads)" be reworded as "workloads (VNFs/CNFs)" as in the first paragraph of the chapter:
> > 
> > The Cloud Infrastructure is the totality of all hardware and software components which build up the environment in which VNFs/CNFs (workloads) are deployed, managed and executed. It is, therefore, inevitable that different workloads would require different capabilities and have different expectations from it.
> > 
> > 1. Change The Figure legend: "Figure 2-1: Infrastructure profiles proposed based on VNFs categorisation." to "Figure 2-1: Infrastructure profiles proposed based on workloads (VNFs/CNFs) categorisation."
> 
> @pgoyal01 due to the fact the here we initially mention VNFC so I used microservices to corresponding to VNFC, but I ok to change all to "workloads (VNFs/CNFs)"

For VNFC-s I would rather use Pods/VM, than microservices as these two are in different abstraction levels. Using workloads (VNFs/CNFs) does not express, that this is a mire fine grained unit, than a whole application.

File: doc/ref_model/chapters/chapter02.md [hunk 1]
```diff
@@ -170,8 +170,8 @@ By trying to sort workloads into different categories based on the requirements
 # 2.3 Cloud Infrastructure Profiles
 
 Based on the above analysis, following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)
-- **Basic**: for VNFCs that can tolerate resource over-subscription and variable latency.
-- **Network Intensive**: for VNFCs that require predictable computing performance, high network throughput and low network latency.
+- **Basic**: for Workloads [VNFCs/Micro-service] that can tolerate resource over-subscription and variable latency.
```
Comment (CsatariGergely) [inline doc/ref_model/chapters/chapter02.md:173]: My 2 cents are "Workoads (Pods/VM-s)"

Comment (ASawwaf) [reply doc/ref_model/chapters/chapter02.md:173]: @CsatariGergely @pgoyal01 , what about ( Pods/VNFc)

Comment (hdamker) [reply doc/ref_model/chapters/chapter02.md:173]: My proposal is to omit the brackets completely. We have clear definition of "Workloads" in Glossary, and there is no need to refer to the very specific ETSI NFV term VNFC here.

**Comment by kedmison** [comment]
> See my proposal to omit the brackets behind Workloads completely and therefore avoid the discussion. Workloads are sufficiently defined within the Glossary.

@ASawwaf can you please take a look at these comments to see how to resolve the requested changes?

**Comment by ASawwaf** [comment]
@kedmison I will go with @hdamker suggestion 

BTW in the glossary : "VMs or Containers "
"Workload: Workload refers to software running on top of compute resources such as VMs or Containers. Most relevant workload categories in context of cloud infrastructure are"

**Comment by ASawwaf** [comment]
@kedmison @CsatariGergely 
can you approve to merge

**Comment by ASawwaf** [comment]
@kedmison @CsatariGergely @rabi-abdel 
can you approve to merge

**Pull request merged**